WSLBlobNFS/wsl2_linux_script.sh

```diff
@@ -122,28 +211,40 @@ function unmount_share ()
     if [[ $startline < $endline ]]; then
         # Remove the share details from smb.conf
         sed -i "$startline,$endline d" /etc/samba/smb.conf
-        echo "Removed SMB share with: $1."
+        secho "Removed SMB share with: $1."
```

Comment:
nit, don't use raw parameters in the function. it's more readable to capture them in an appropriately named local variable at the start of the function and then use that name

local smbshare=$1

and use $smbshare in all places

Reply:
github doesn't allow me to add comments on lines which have not changed in the PR.
so commenting here

L191, did you mean this

 [[ $foundshare == "true" && "$line" =~ \\[.*\\] ]];

also L199
"$line" =~ path.**

in general use =~ for regexp matching